hassio-google-drive-backup/backup/static/logs.html

```diff
@@ -32,18 +32,10 @@
     <script type="text/javascript">
         toasted = false;
         running = false;
-        function resize() {
-            var offset = 0;
-            if (!$('#hello_card').data("closed")) {
-                offset = $("#hello_card").height();
-            }
-            $("#log_container").css("height", ($(window).height() - offset - 50) + "px");
-        }
```

Comment:
By using `style="height: 100%; display: flex; flex-direction: column"` on the wrapper div, we don't need to calculate the height with javascript on resize; CSS does it for us :)

Reply:
Thats great.  You'll probably notice this as you run around the codebase, but my html/javascript is pretty weak.
I'm more of a "backend" kind of programmer, and all my js/css/etc is basically just hacked together until it works :)

Reply:
This is not a problem at all :) If I happen to find other hacks that could be cleaned up, I will sure check them out.